docs/model-dev-guide/model-management/checkpoints.rst

```diff
@@ -40,68 +40,68 @@ configuration file will result in checkpoints being sorted by the loss metric in
      smaller_is_better: true
 
 The following snippet of Python code can be run after the specified experiment has generated a
-checkpoint. It returns an instance of :class:`~determined.experimental.client.Checkpoint`
-representing the checkpoint that has the best validation metric.
+checkpoint. It gets a list of sorted :class:`~determined.experimental.client.Checkpoint` instances
+associated with the experiment and selects the one with the best validation metric.
 
 .. code:: python
 
    from determined.experimental import client
 
-   checkpoint = client.get_experiment(id).top_checkpoint()
+   checkpoint = client.get_experiment(id).list_checkpoints()[0]
 
 Checkpoints can be sorted by any metric using the ``sort_by`` keyword argument, which defines which
-metric to use, and ``smaller_is_better``, which defines whether to sort the checkpoints in ascending
-or descending order with respect to the specified metric.
+metric to use, and ``order_by``, which defines whether to sort the checkpoints in ascending or
+descending order with respect to the specified metric.
 
 .. code:: python
 
-   from determined.experimental import client
+   from determined.experimental import checkpoint, client
 
-   checkpoint = (
-       client.get_experiment(id).top_checkpoint(sort_by="accuracy", smaller_is_better=False)
+   checkpoints = (
+       client.get_experiment(id).list_checkpoints(
+           sort_by="accuracy",
+           order_by=checkpoint.CheckpointOrderBy.DESC
+       )
    )
 
-You may also query multiple checkpoints at the same time using the
-:meth:`~determined.experimental.client.ExperimentReference.top_n_checkpoints` method. Only the
-single best checkpoint from each trial is considered; out of those, the checkpoints with the best
-validation metric values are returned in sorted order, with the best one first. For example, the
-following snippet returns the top five checkpoints from distinct trials of a specified experiment.
+You may also query checkpoints by preset checkpoint parameters, as defined by
+:class:`~determined.experimental.checkpoint.CheckpointSortBy`. The following code snippet fetches
+all checkpoints for the experiment sorted by the checkpoint's trial ID in descending order.
```

Comment:
To sort checkpoints using preset checkpoint parameters, use the :class:`~determined.experimental.checkpoint.CheckpointSortBy` class. The example below fetches all checkpoints for an experiment, sorting them by trial ID in descending order.